.github/workflows/deploy.js.yml

```diff
@@ -5,7 +5,7 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ automatic_deploy ]
```

Comment:
Czy to znaczy, że teraz, żeby zrobić deploya, robimy PR do brancha `automatic_deploy`?

Reply:
To jest testowe, bo ogólnie to co tutaj pisze, że action uruchamia się tylko wtedy gdy robiony jest push na tego brancha. Domyślnie będzie to na masterze, ale chcę zobaczyć czy deploy się poprawnie wykona